frontends/mit-open/src/page-components/SearchDisplay/SearchDisplay.tsx

```diff
@@ -680,7 +681,15 @@ const SearchDisplay: React.FC<SearchDisplayProps> = ({
                 <Pagination
                   count={getLastPage(data?.count ?? 0)}
                   page={page}
-                  onChange={(_, newPage) => setPage(newPage)}
+                  onChange={(_, newPage) => {
+                    setPage(newPage)
+                    setTimeout(() => {
+                      scrollHook.current?.scrollIntoView({
+                        block: "center",
+                        behavior: "smooth",
+                      })
+                    }, 0)
```

Comment:
Out of curiosity... why wrap this in setTimeout?

Reply:
Despite some investigating, all I can say for sure is that it doesn't work without it. Likely some nuance around the dom element not being available during the old render cycle.